src/leaflet_property.js

```diff
@@ -1,6 +1,7 @@
 import L from 'leaflet';
 
 export const defaultMarkerColor = '#A53434';
+export const selectMarkerColor = '#FF6347';
```

Comment:
would a proper tense of the verb "to select" make for a clearer variable name? ie selectedMarkerColor (the colour of the marker which has been selected)